packages/containers/psammead-timestamp-container/src/relativeTimestamp.js

```diff
@@ -0,0 +1,36 @@
+const constructTimestamp = timeDiffMins => {
+  let timeDiff;
+  let unit;
+
+  if (timeDiffMins >= 1440) {
```

Comment:
could we store 1440 in a variable like `twentyFourHours` or something similar so it's easy to see what the significance of 1440 is without googling?